oaknational/Oak-Web-Application
# fix: only combine unit listing pages for ks1-4 maths [LESQ-949]

## Description

Music year: 2003

- EYFS was being excluded in the logic for combining new and legacy maths, as there is only legacy content
- this includes EYFS for maths subject only and doesn't combine counts

## Issue(s)

Fixes #
Maths EYFS gone missing from the site

## How to test

1. Go to https://deploy-preview-2559--oak-web-application.netlify.thenational.academy/teachers/key-stages/early-years-foundation-stage/subjects
3. You should see a card for maths
4. Click through and the counts should be correct
5. There should be no 'new' tag on any of the content

## Screenshots

How it used to look (delete if n/a):
<img width="600" alt="Screenshot 2024-07-03 at 08 34 33" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/f4768f60-496c-4019-8bd3-ee9a8455f70d">

How it should now look:
<img width="600" alt="Screenshot 2024-07-03 at 08 34 44" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/5719a76e-c25d-4a20-a8c3-553aba3b2467">
